extensions/resteasy-classic/rest-client/runtime/src/main/java/io/quarkus/restclient/runtime/RestClientRecorder.java

```diff
@@ -43,6 +43,7 @@ public InjectorFactory getInjectorFactory() {
         registerProviders(clientProviderFactory, useBuiltIn, providersToRegister, contributedProviders);
         RestClientBuilderImpl.setProviderFactory(clientProviderFactory);
         ResteasyClientBuilderImpl.setProviderFactory(clientProviderFactory);
+        ResteasyProviderFactory.setInstance(clientProviderFactory);
```

Comment:
Won't it also override the server defined one?

Reply:
Didn't realize that, now it's set only when RESTEasy classic is absent. Thank you